.github/workflows/test-and-publish-nightly.yml

```diff
@@ -35,6 +35,10 @@ jobs:
               display: linux-x64
               name: ubuntu-latest
             image: public.ecr.aws/async-profiler/asprof-builder-x86:latest
+          - runson:
+              display: alpine
```

Comment:
Will artifacts from this build be published in a release? They should not.

Reply:
I thought Alpine should have its own artifacts, fixed in 91e17e201827dd96750e1a78fa8cf61fe1f8a0df